Dropdown.vue

```diff
@@ -5,7 +5,7 @@
           <span class="caret"></span>
         </li>
 
-        <li @click="toggleMenu()" class="dropdown-toggle" v-if="selectedOption.name === undefined">
+        <li @click="toggleMenu()" class="dropdown-toggle placeholder" v-if="selectedOption.name === undefined">
```

Comment:
I think `placeholder` is a very generic class name, which could clash with other users current projects so if we could change it so something a bit less generic such as `dropdown-toggle-placeholder` that probably would be a bit better.

Reply:
I agree.

Class name changed as suggested, in subsequent commit to PR.